packages/@aws-cdk/aws-rds/lib/instance.ts

```diff
@@ -111,26 +111,6 @@ export interface DatabaseInstanceAttributes {
  * A new or imported database instance.
  */
 export abstract class DatabaseInstanceBase extends Resource implements IDatabaseInstance {
-  /**
-   * Import an existing database instance.
-   */
-  public static fromDatabaseInstanceAttributes(scope: Construct, id: string, attrs: DatabaseInstanceAttributes): IDatabaseInstance {
-    class Import extends DatabaseInstanceBase implements IDatabaseInstance {
-      public readonly defaultPort = ec2.Port.tcp(attrs.port);
-      public readonly connections = new ec2.Connections({
-        securityGroups: attrs.securityGroups,
-        defaultPort: this.defaultPort,
-      });
-      public readonly instanceIdentifier = attrs.instanceIdentifier;
-      public readonly dbInstanceEndpointAddress = attrs.instanceEndpointAddress;
-      public readonly dbInstanceEndpointPort = attrs.port.toString();
-      public readonly instanceEndpoint = new Endpoint(attrs.instanceEndpointAddress, attrs.port);
-      public readonly engine = attrs.engine;
-      protected enableIamAuthentication = true;
-    }
-
-    return new Import(scope, id);
-  }
```

Comment:
Since our backwards-compatibility checker is complaining, let's just leave this method as-is, deprecate it, and delegate in its body to `DatabaseInstance. fromDatabaseInstanceAttributes`.

Reply:
Done!